Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan.yml

```diff
@@ -2564,138 +791,165 @@ tasks:
-          root: inputs.IP
-          filters:
-          - - operator: in
-              left:
+              equals: {}
+              lhs:
                 value:
-                  simple: inputs.IP
+                  simple: inputs.AutoContainment
                 iscontext: true
-              right:
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs:
+                value:
+                  simple: "True"
+              rhsB: {}
+              then:
                 value:
-                  simple: Block Indicators.Answers.1
+                  simple: inputs.Username
                 iscontext: true
-              ignorecase: true
-          transformers:
-          - operator: uniq
-      InputEnrichment:
-        simple: "False"
-      MD5:
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 900,
+          "y": 170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "145":
+    id: "145"
+    taskid: 8bd87a2a-710a-479b-8356-540bcdf010fd
+    type: playbook
+    task:
+      id: 8bd87a2a-710a-479b-8356-540bcdf010fd
+      version: -1
+      name: Containment Plan - Isolate Device
+      playbookName: Containment Plan - Isolate Device
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "148"
+    scriptarguments:
+      EndpointHostName:
         complex:
-          root: inputs.FileHash
-          filters:
-          - - operator: stringHasLength
-              left:
+          root: inputs.AutoContainment
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
                 value:
-                  simple: inputs.FileHash
-                iscontext: true
-              right:
+                  simple: lhs==rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
                 value:
-                  simple: "32"
-          - - operator: in
-              left:
+                  simple: Which containment actions would you like to perform?.Answers.0
+                iscontext: true
+              equals: {}
+              lhs:
                 value:
-                  simple: inputs.FileHash
+                  simple: inputs.AutoContainment
                 iscontext: true
-              right:
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs:
                 value:
-                  simple: BlockIndicatorsList
+                  simple: "True"
+              rhsB: {}
+              then:
+                value:
+                  simple: Core.Endpoint.endpoint_name
```

Comment:
Please use a playbook input.

Reply:
There is no input for the endpoint name.